.storybook/preview-head.html

```diff
@@ -0,0 +1,9 @@
+<script>
+    window['__HEADERS__'] = {
+        // This is needed because perseus uses jquery,
+        // and jquery depends indirectly on khan-fetch,
+        // which has a toplevel check for this permissions
+        // header.
+        "X-KA-has-any-permissions": "true",
+    };
```

Comment:
I don't think we need this.  `pereus-editor` is the only thing that uses `jquery` and that package uses a standard build of it as a devDep/peerDep.